tests/test_configure_feedstock.py

```diff
@@ -578,30 +592,42 @@ def test_secrets(py_recipe, jinja_env):
         )
 
 
-def test_migrator_recipe(recipe_migration_cfep9, jinja_env):
+def test_migrator_recipe(recipe_migration_cfep9, jinja_env, request):
     configure_feedstock.render_azure(
         jinja_env=jinja_env,
         forge_config=recipe_migration_cfep9.config,
         forge_dir=recipe_migration_cfep9.recipe,
     )
 
+    ci_support_filename = "linux_64_python2.7.yaml"
+    conda_build_param = request.node.callspec.params["config_yaml"]
+    if conda_build_param == "rattler-build":
+        ci_support_filename = "linux_64_python2.7zlib1000.yaml"
```

Comment:
Why this change?

Reply:
rattler-build expose all variants that are used during the rendering so because we generate filenames using them - for rattler-build they are little longer and contains all of them

Reply:
Can we fix that in rattler-build or `ratller-build` or `rattler-build-conda-compat`. Otherwise this would include a lot of unnecessary information.

Reply:
> Can we fix that in rattler-build or `ratller-build` or `rattler-build-conda-compat`. Otherwise this would include a lot of unnecessary information.

Yes - I can change it

Reply:
done - it will generate the same name as conda-build does

Reply:
until the new version of rattler-build-conda-compat will be picked up by solver the tests will fail temporary